tests/edge-to-edge/server/app.spec.ts

```diff
@@ -166,21 +172,32 @@ describe( 'Termbox SSR', () => {
 
 			const $dom = getDomFromMarkup( response.text );
 
-			expect( $dom.querySelectorAll( '.wikibase-termbox' ).length ).toBe( 1 );
+			const termboxVersions = $dom.querySelectorAll( '.wikibase-termbox' );
+
+			// 3 hard coded languages currently in src/store/language/actions.ts
+			expect( termboxVersions.length ).toBe( 3 );
```

Comment:
Maybe Object.keys( response.wbcontentlanguages ).length instead?

Reply:
I intentionally hard-coded this to break as soon as the list is dynamized (e.g. https://phabricator.wikimedia.org/T209288).